backstop/scenarios/button.js

```diff
@@ -21,8 +21,13 @@ module.exports = [
   }),
   scenario('Type Split Menu', {
     selectors: ['#demo-split-menu'],
+    misMatchThreshold: 0.03,
```

Comment:
Why these are needed here?

Reply:
The tests were failing without it, even with no visible change. I removed threshold and updated images, as the images were already changing.

For the ideas button I left it as I didn't want to update image

Reply:
Maybe update ideas button too and we can just remove increased threshold from there.

Reply:
ok